install_config/master_node_configuration.adoc

```diff
@@ -1199,16 +1199,17 @@ The advanced audit feature provides several improvements over the
 xref:master-node-config-audit-config[basic audit functionality], including
 fine-grained events filtering and multiple output back ends.
 
-To enable the advanced audit feature, provide the following values in the
-`openshift_master_audit_config` parameter:
+To enable the advanced audit feature using a policy file, provide the following values in the
```

Comment:
s/feature using a policy file,provide/feature, you create an audit policy file and specify
So it starts with:
> To enable the advanced audit feature, you create an audit policy file and specify...

Reply:
I has modified! Thank you for your kind point!